hyperdrive-release_spark-3/pom.xml

```diff
@@ -74,6 +74,8 @@
                                         <Implementation-Version>${buildNumber}</Implementation-Version>
                                     </manifestEntries>
                                 </transformer>
+                                <!-- This allows to support several library data sources for Spark by referring to them by short name. -->
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
```

Comment:
Interesting. Didn't know about this option. Probably I can get rid of `<root>/driver/src/main/resources/META-INF/services/org.apache.spark.sql.sources.DataSourceRegister` as well.